Unity3D/Assets/RosSharp/Scripts/ImageSubscriber.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+
+using UnityEngine;
+
+namespace RosSharp.RosBridgeClient
+{
+
+    [RequireComponent(typeof(RosConnector))]
+    public class ImageSubscriber : MonoBehaviour
+    {
+        public GameObject Texture;
+        private ImageManager imageManager;
+        private RosSocket rosSocket;
+        public string topic = "/image_raw";
+        public int UpdateTime = 1;
+
+        public void Start()
+        {
+            rosSocket = transform.GetComponent<RosConnector>().RosSocket;
+            rosSocket.Subscribe(topic, "sensor_msgs/Image", updateTexture, UpdateTime);
+
+            imageManager = Texture.GetComponent<ImageManager>();
+        }
+
+        private void updateTexture(Message message)
+        {
+            SensorImage sensorImage = (SensorImage) message;
+            imageManager.UpdateTexture(sensorImage);
```

Comment:
Please replace line 27 and 28 by:
imageManager.UpdateTexture((SensorImage) message);